warehouse/templates/includes/current-user-indicator.html

```diff
@@ -29,6 +29,10 @@
           Admin
         </a>
         {% endif %}
+        <a class="dropdown__link" href="{{ request.route_path('accounts.profile', username=request.user.username) }}">
+          <i class="fa fa-cube" aria-hidden="true"></i>
+          Your Profile
```

Comment:
Should I rename this to "Your Public Profile"?

Reply:
If it fits, that would be better @berkerpeksag

Reply:
@berkerpeksag if "Your public profile" does not fit, I think "Public profile" would suffice.  IMO, it's clearer than "Your profile"